### ISSUE TITLE
install & configuration speed ups

### ISSUE THREAD

**Issue by schmto** [nmsprime/nmsprime#556]
1. configfile: auto replace SNMP communities with Global Settings

Instead of manually changing the community for every ISP configfile, we should introduce something
like this:
```

SnmpMibObject docsDevNmAccessCommunity.10 String "{provbase.ro_community.0}" ;
SnmpMibObject docsDevNmAccessCommunity.20 String "{provbase.rw_community.0}" ;
```

which should be replaced in the config files for example to:
```

SnmpMibObject docsDevNmAccessCommunity.10 String "public" ;
SnmpMibObject docsDevNmAccessCommunity.20 String "private" ;
```

Maybe see code
https://github.com/nmsprime/nmsprime/blob/13afce73dfc79262912affd6e2580a8c5813d14c/modules/ProvBase/Entities/Configfile.php#L200

2. product IDs for invoice concatenation of invoices for postal delivery is actually set manually via file - storage/app/config/billingbase/post-invoice-product-ids. This should be solved somehow in the GUI? Like a checkbox for special products

3. cp storage/app/config/billingbase/bic_de.csv
This should be solved automatically, for example by loading this file from
support.nmsprime.com. For inspiration see: https://github.com/nmsprime/nmsprime/blob/3199fdba09a1f2a313976b87044e1c456759d242/modules/Dashboard/Http/Controllers/DashboardController.php#L374

4. Add Read Only Role by Default

**Comment by cschra** [comment]
to 4. 
There are 3 Roles by default:
- Admin -> can do everything
- Support -> can do everything except changing roles
- Guest -> can view everything, but cannot modify anything (read only)

If this is a clarity issue, the guest role can be renamed, but a read only role exists as default.

**Comment by royschn** [comment]
1. fixed in 3c4fda3

**Comment by royschn** [comment]
2. fixed in 91263d8
